libs/platform/core/src/services/page-meta/page-meta.service.ts

```diff
@@ -3,7 +3,10 @@ import { ElementAttributes, ElementDefinition } from './page-meta.model';
 export const PageMetaService = 'oryx.PageMetaService';
 
 export interface PageMetaService {
-  add(definitions: ElementDefinition | ElementDefinition[]): void;
+  add(
+    definitions: ElementDefinition | ElementDefinition[],
+    forceCreation?: boolean
```

Comment:
Can you explain, why we need it?

Reply:
Right now if we have the same element in the head we do nothing. Force creation forcibly removes this element and recreate new one (e.g load page - add title => go to the product - recreate title attribute)

Reply:
but we just should update the text, we don't have to recreate an attribute?